pom.xml

```diff
@@ -50,13 +50,15 @@
         <!-- EAP 6.4.x -->
         <version.eap>7.5.6.Final-redhat-2</version.eap>
         <!-- Perfectus managed versions -->
-        <version.fuse>6.2.1.redhat-084</version.fuse>
+        <version.fuse>6.3.0.redhat-045</version.fuse>
         <version.fuse.eap>${version.fuse}</version.fuse.eap>
         <version.fuse.patch>1.6.2</version.fuse.patch>
-        <version.org.apache.karaf>2.4.0.redhat-621084</version.org.apache.karaf>
+        <version.org.apache.karaf>2.4.0.redhat-630045</version.org.apache.karaf>
         <version.camelsap>${version.fuse}</version.camelsap>
-        <version.switchyard>2.0.1-redhat-SNAPSHOT</version.switchyard>
-        <version.wildfly.camel>2.3.0.redhat-621084</version.wildfly.camel>
+        <version.switchyard>2.1.0-redhat-SNAPSHOT</version.switchyard>
+        <version.wildfly.camel>2.4.0.redhat-630045</version.wildfly.camel>
+        <version.apache.camel>2.16.1.redhat-630045</version.apache.camel>
```

Comment:
This is upgraded by the perfectus build. And the changes are commited in the respective tag.

When a new release of fuse-bxms-integ is done, then it would be nice to upgrade these values as well as maintain the compatibility with previous versions.